examples/wearable/UIComponents/js/circle-helper.js

```diff
@@ -17,12 +17,15 @@ document.addEventListener("tauinit", function () {
 				pageId = page.id,
 				list;
 
+			tau.util.rotaryScrolling.enable(event.target.firstElementChild);
```

Comment:
This patch adding redundant support for rotation for content & lists. The lists have own support for rotary event. This PR may cause issues hard to predict.